openapi/paths/v2/projects/{id}/members.js

```diff
@@ -2,22 +2,31 @@ const _ = require( "lodash" );
 const Joi = require( "joi" );
 const transform = require( "../../../../joi_to_openapi_parameter" );
 const ProjectsController = require( "../../../../../lib/controllers/v2/projects_controller" );
-const observationsSearchSchema = require( "../../../../schema/request/projects_members" );
+const projectsMembersSchema = require( "../../../../schema/request/projects_members" );
 
 module.exports = sendWrapper => {
   async function GET( req, res ) {
     const results = await ProjectsController.members( req );
     sendWrapper( req, res, null, results );
   }
 
-  const parameters = _.map(
-    observationsSearchSchema.$_terms.keys,
-    child => transform( child.schema.label( child.key ) )
+  const parameters = [
+    transform(
+      Joi.number( ).integer( )
+        .label( "id" )
+        .meta( { in: "path" } )
+        .required( )
+        .description( "A single ID" )
```

Comment:
Shouldn't this be "A single ID or a comma-separated list of IDs"?

Reply:
Unfortunately not, /projects/{id}/members endpoint is only supporting a single project ID, unlike the other projects endpoints.